_includes/releases/v22.1/v22.1.14.md

```diff
@@ -0,0 +1,62 @@
+
+<h3 id="v22-1-14-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where, in a cluster with nodes running both [v22.2](v22.2.html) and v22.1, range replica changes could sometimes fail on v22.1 leaseholders with the error `change replicas of r47 failed: descriptor changed: [expected] != [actual]`, without any apparent differences between the listed descriptors. Continuing to upgrade all nodes to v22.2 or rolling all nodes back to v22.1 would resolve this issue. [#94841][#94841]
+- It is now possible to run [`cockroach version`](../v22.2/cockroach-version.html) and [`cockroach start`](../v22.2/cockroach-start.html) (and possibly other sub-commands) when the user running the command does not have permission to access the current working directory. [#94926][#94926]
+- Fixed a bug where [`CLOSE ALL`](../v22.1/sql-grammar.html#close_cursor_stmt) would not respect the `ALL` flag and would instead attempt to close a cursor with no name. [#95440][#95440]
+- Fixed a crash that could happen when formatting a tuple with an unknown type. [#95422][#95422]
+- Fixes a bug where a [database restore](../v22.1/restore.html) would not grant `CREATE` and `USAGE` on the public schema to the public role. [#95537][#95537]
+- Fixed a bug where [`pg_get_indexdef`](../v22.1/functions-and-operators.html) didn't show the expression used to define an expression-based index, as well as a bug where the function was incorrectly including columns stored by the index. [#95585][#95585]
+- Fixed a bug where a DNS lookup was performed during gossip remote forwarding while holding the gossip mutex. This could cause processing stalls if the DNS server was slow to respond. [#95441][#95441]
+- Fixed a bug where [`RESTORE SYSTEM USERS`](../v22.1/restore.html#restoring-users-from-system-users-backup) would fail to restore role options. [#95295][#95295]
+- Reduce register session, deregister session, and session cancel query contention. [#95654][#95654]
+- Fixed a bug where a backup of keys with many revisions would fail with `pebble: keys must be added in order`. [#95446][#95446]
+- Fixed the `array_to_string` [built-in function](../v22.1/functions-and-operators.html) so that nested arrays are traversed without printing `ARRAY` at each nesting level. [#95844][#95844]
+- Fixed a bug that caused ranges to remain without a leaseholder in cases of asymmetric network partitions. [#95237][#95237]
+- Fixed a bug where [`COPY`](../v22.1/copy-from.html) into a column with collated strings would result in an error similar to `internal error: unknown type collatedstring`. [#96039][#96039]
+- Fixed a bug where spurious transaction restarts could occur when validating a [`FOREIGN KEY`](../v22.1/foreign-key.html) in the same transaction where the referenced table is modified. If the transaction was running at `PRIORITY HIGH`, deadlocks could occur. [#96124][#96124]
```

Comment:
suggest making 'transaction restarts' a link to https://www.cockroachlabs.com/docs/stable/common-errors.html#restart-transaction

suggest making `PRIORITY HIGH` a link to https://www.cockroachlabs.com/docs/v22.1/transactions#transaction-priorities